pkg/pointerdb/pointerdb.go

```diff
@@ -30,7 +30,7 @@ var (
 // ListPageLimit is the maximum number of items that will be returned by a list
 // request.
 // TODO(kaloyan): make it configurable
-const ListPageLimit = 1000
+const ListPageLimit = storage.LookupLimit
```

Comment:
I wonder if we still need this const here and the check in the List method, if this limit check is now handled in the DB.

Reply:
I was thinking about it, I also thought about how to configure it. There are a few roads we could take:

1. Remove the limit from KeyValueStore and enforce it at service level.
2. Add a wrapper around KeyValueStore that enforces such limits.

As usual with these things, since the cost of defensive programming here isn't high, it can be duplicated in both places.

Reply:
Ok, I'll remove it for now and let's revisit the problem when making it configurable.